apiserver/embeddedcommands.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package apiserver
```

Comment:
Is there a reason to have this in the apiserver package rather than a subpackage? If it was a subpackage it could be more easily brought into JIMM.

Reply:
I had it in a subpackage originally. Then when the tests were best written in the apiserver_test.go file I thought it a little weird to have a subpackage with just one go file. But I can move back to a subpackage.